crates/service_utils/src/helpers.rs

```diff
@@ -401,3 +411,91 @@ pub fn parse_config_tags(
         }
     }
 }
+
+pub async fn execute_webhook_call<T>(
+    webhook_object: Value,
+    payload: T,
+    config_version_opt: &Option<String>,
+    event: WebhookEvent,
+) -> Result<(), AppError>
+where
+    T: Serialize,
+{
+    match serde_json::from_value::<Webhook>(webhook_object) {
```

Comment:
@mahatoankitkumar can we use `?` to unwrap the object earlier, and avoid this nesting?